style.css

```diff
@@ -214,7 +214,7 @@ nav ul li a:hover::after{
 }
 .portfolio-col img{
     width: 100%;
-    height: auto;
```

Comment:
the height value is 100% when i run it locally. But when i publish the styles.css to github, it reverts back to auto. I have no idea what am doing wrong.